### DISCUSSION TITLE
Do you mind if I start a SaaS with Fooocus as backend?

### DISCUSSION THREAD

**Discussion by fyepi** [lllyasviel/Fooocus#1639] [General]
Hello,

I work for a pretty big company's internal IT. I have deployed Fooocus as internal SD backend and our creativity and marketing guys love it.

I am thinking of setting up a SaaS platform with Fooocus as backend to provide API services, with a rich varieties of basemodels and loras.

According to Fooocus license it is OK to do so but obviously it is the best to get blessing from Fooocus maintainers.  If by any chance, the SaaS makes money, I would be very happy to share profit with the maintainers in whatever way appropriate.

For obvious reason I registered a new Github account for this plan.

Thanks again for this great product.

**Comment by danhigham** [comment]
The license is GNU GPL and I am pretty sure that would be allowed under that license, perhaps read up on the license specifically.

**Reply by fyepi** [reply]
Yes I thought so too. It is just that I want to check with them if they have any concerns.

**Comment by Aeschbach** [comment]
Hello 
I am a designer and design educator (not a developer). In this role I am teaching picture generative AI tools (Dalee, Midjourney, Adobe Firefly and Stable Diffusion) in two schools. For this I set up Fooocus a Hugging-Face-Space online for my students. Works ok as MVP. but not good enough to roll it out in a business context. I am working on developing this in to a service for different institutional clients.
Are you interested to collaborate on this?
